frontend/src/components/WorkspaceSettings/Billing/BillingPlanCard/BillingPlanCardDialog.tsx

```diff
@@ -39,7 +39,7 @@ const BillingPlanCardDialog = (props: BillingPlanCardDialogProps) => {
 
     const response = await updateSubscriptionApiV1BillingSubscriptionSubscriptionIdPut(subscriptionId, {
       price_id: priceId,
-      quantity: quantity
+      quantity: quantity - 1
```

Comment:
didn't we say we're doing that on the BE side?

Reply:
I wanted but unfortunately no